lib/rubocop/ast/node/mixin/method_dispatch_node.rb

```diff
@@ -8,6 +8,8 @@ module MethodDispatchNode
       extend NodePattern::Macros
       include MethodIdentifierPredicates
 
+      ACCESS_MODIFIERS = %i[public protected private module_function].freeze
```

Comment:
I don't think they belong in this PR. They weren't covered by any of the cops that I updated and adding support for them would require adding other public API to the send nodes.